### PR TITLE
CHANGELOG: release RC11

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+   CalculateAndSetSignature/VerifyIDSignature, harmonizing this API with other
+   structures (#513)
+ * System role is ignored in the EACL matcher now (#515)
+
+Improvements:
+ * extended integration tests (#467)
+ * new well-known "Version" node attribute support (#490)
+ * documentation and examples (#495, #511, #507, #509)
+ * exported AttributeExpirationEpoch (#504)
+ * methods to with with user's container and object attributes (#505, #508)
```
Comment (carpawell) [inline CHANGELOG.md:32]:
```suggestion
 * methods to work with user's container and object attributes (#505, #508)
```

or what is here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+Behaviour changes:
+ * ObjectRangeInit and ObjectRangeInit unification between Client and Pool
+   (#491)
+ * removal of all deprecated Pool APIs (#491)
+ * ObjectHead method now returns header instead of intermediate struct (#497)
+ * ObjectListReader now returns error from Read instead of bool flag
+ * Object's Sign/VerifySignature methods replacing
+   CalculateAndSetSignature/VerifyIDSignature, harmonizing this API with other
+   structures (#513)
+ * System role is ignored in the EACL matcher now (#515)
```
Comment (carpawell) [inline CHANGELOG.md:25]:
```suggestion
 * System role is ignored in the eACL matcher now (#515)
```
seems like we prefer this spelling usually. not critical

File: CHANGELOG.md (hunk 1 above)
Comment (roman-khimov) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (roman-khimov) [reply CHANGELOG.md]: OK, fixed.

**Review by carpawell — APPROVED** [review]
😭 https://github.com/nspcc-dev/.github/issues/3 😭

**Pull request merged**